manifest.toml

```diff
@@ -0,0 +1,100 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/manifest.v2.schema.json
+
+packaging_format = 2
+
+id = "weblate"
+name = "Weblate"
+description.en = "Translation platform using Git and Python"
+description.fr = "Plateforme de traduction utilisant Git et Python"
+
+version = "5.3.1~ynh1"
+
+maintainers = ["YunoHost Contributors"]
+
+[upstream]
+license = "AGPL-3.0"
+website = "https://weblate.org"
+demo = "https://hosted.weblate.org/"
+admindoc = "https://docs.weblate.org/"
+userdoc = "https://docs.weblate.org/"
+code = "https://github.com/WeblateOrg/weblate"
+cpe = "cpe:2.3:a:weblate:weblate"
+
+[integration]
+yunohost = ">= 11.2.0"
+architectures = "all"
+multi_instance = true
+ldap = false
+sso = false
+
+disk = "1G"
+ram.build = "3G"
+ram.runtime = "3G"
```

Comment:
Wow, thats’ huge :o

Reply:
That’s what is said in Weblate’s installation documentation. I think it’s just an info in Yunohost or does it prevent installation on lower hardware?

Reply:
Nah, it's not a blocker. It's just an indication, and maybe a warning that the user needs to dismiss.